charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml

```diff
@@ -174,10 +174,10 @@ spec:
             timeoutSeconds: 1
           resources:
             limits:
-              cpu: 1350m
```

Comment:
Interesting, did it fail?

Reply:
Yes :)

Reply:
```
49s (x17 over 63s)      Warning   FailedCreate             StatefulSet/medic-y-2024-cw-45-0a3701a8-benchmark-zeebe                     create Pod medic-y-2024-cw-45-0a3701a8-benchmark-zeebe-2 in StatefulSet medic-y-2024-cw-45-0a3701a8-benchmark-zeebe failed error: Pod "medic-y-2024-cw-45-0a3701a8-benchmark-zeebe-2" is invalid: spec.containers[0].resources.requests: Invalid value: "1700m": must be less than or equal to cpu limit of 1350m
```

Reply:
I guess we should have an integration test